actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -87,6 +87,7 @@ module ClassMethods
       #
       # * <tt>:only/:except</tt> - Passed to the <tt>before_action</tt> call. Set which actions are verified.
       # * <tt>:if/:unless</tt> - Passed to the <tt>before_action</tt> call. Set when actions are verified.
+      # * <tt>:prepend</tt> - ?
```

Comment:
I think `Passed to the <tt>before_action</tt> call.` should be skipped here since it is not commonly used option. Any idea how to describe it here in few words?